src/pages/Mypage/UserInfo/UserInfo.js

```diff
@@ -1,32 +1,64 @@
 import React from 'react';
-import { SUB_TITLE } from '../SUB_TITLE';
-import BASE_URL from '../../../config';
+import { API } from '../../../config';
 import { Avatar, Card, CardMedia } from '@mui/material';
 import BookmarkIcon from '@mui/icons-material/Bookmark';
 import useFetch from '../../../hooks/useFetch';
 import * as S from './UserInfo.style';
+import { useEffect } from 'react';
+import { useState } from 'react';
```

Comment:
```suggestion
import React { useState,useEffect } from 'react';
```

같은패키지에서 import한 hooks는 한줄로 쓸 수 있어요~